js/main.js

```diff
@@ -169,41 +206,23 @@ YUI.add(
 					}
 				},
 
-				handleCurrentCommandSelect: function(event) {
-					var instance = this;
-
-					var xmlLog = instance.get(STR_XML_LOG);
-
-					var currentTargetAncestor = event.currentTarget.ancestor();
-
-					if (currentTargetAncestor) {
-						instance._parseCommandLog(currentTargetAncestor);
-
-						var functionLinkId = currentTargetAncestor.attr(ATTR_DATA_FUNCTION_LINK_ID);
-
-						var linkedFunction = xmlLog.one('.line-group[data-functionLinkId="' + functionLinkId + '"]');
-
-						instance._displayNode(linkedFunction);
-
-						instance._selectCurrentScope(linkedFunction);
-					}
-				},
-
 				handleFullScreenImageClick: function(event) {
 					var instance = this;
 
 					var fullScreenImageContainer = A.one('#fullScreenImage');
 
 					if (fullScreenImageContainer) {
+						var src;
+
 						var fullScreenImage = fullScreenImageContainer.one('#image');
 
 						if (fullScreenImage) {
 							var currentTarget = event.currentTarget;
 
-							var src = currentTarget.attr(STR_SRC);
+							src = currentTarget.getData(STR_SRC);
 
 							if (src) {
-								fullScreenImage.attr(STR_SRC, src);
+								fullScreenImage.getData(STR_SRC, src);
```

Comment:
This one should be changed to `.setData()`, since you're changing it (`.attr()` worked as both a getter and setter, but you need to explicitly call `get/set` for the data methods.